ui/app/AppLayouts/Communities/controls/ExtendedDropdownContent.qml

```diff
@@ -127,14 +127,33 @@ Item {
                     AnyOf {
                         enabled: root.showAllTokensMode
 
+                        /*
+                        // this category is not used
                         ValueFilter {
                             roleName: "category"
                             value: TokenCategories.Category.Own
+                        }*/
+
+                        AllOf {
+                            ValueFilter {
+                                roleName: "category"
+                                value: TokenCategories.Category.General
+                            }
+                            ValueFilter {
+                                roleName: "communityId"
+                                value: ""
+                            }
```

Comment:
Is it possible to have general category with a community assigned?

Reply:
Yes, For eg.
![image](https://github.com/status-im/status-desktop/assets/61889657/2727c83d-4a5c-463d-b899-9d5bb44de9af)
I think they are there because they are in the wallet.... eg. I see TUTAJD in the wallet but I do not see ASSX :/